### PR TITLE
Lauren hackteam6

### PR DESCRIPTION
**Pull request by pecheylauren02**

### CODE REVIEW

File: FRONTEND/scripts/about.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+document.getElementById("button").onclick = function () {
```
Comment (sean-meade) [inline FRONTEND/scripts/about.js:1]: Actually, could you give the id a more descriptive name we might decide to add more buttons down the line

File: FRONTEND/scripts/about.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    console.log("Testing");
```
Comment (sean-meade) [inline FRONTEND/scripts/about.js:2]: remove the console.log

File: FRONTEND/scripts/about.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        moreText.classList.remove("hidden");
+
+        // Change text of the button
+        button.innerHTML = "Read less";
+    }
+
+    
+}
+
+console.log("Test 2");
```
Comment (sean-meade) [inline FRONTEND/scripts/about.js:30]: remove the console.log

Comment (pecheylauren02) [reply FRONTEND/scripts/about.js]: Alright I have done all of this! Changed the button ids and removed the console logs

File: FRONTEND/scripts/about.js (hunk 2 above)
Comment (pecheylauren02) [reply FRONTEND/scripts/about.js]: Done

**Pull request closed without merging**